### ISSUE TITLE
Psvita mame2010

### ISSUE THREAD

**Issue by itiloo** [libretro/mame2010-libretro#150]
Hi,
Is this version available por ps vita?.
Than you so much.

**Comment by hizzlekizzle** [comment]
No, I believe Vita struggles to run mame2003, which is significantly faster than 2010.

**Comment by itiloo** [comment]
I have mame2003 plus in psvita with retroarch and its OK for me.
Could the 2010 version be compiled to at least try whoever wants to test it?
Thank you so much
